VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -72,6 +73,8 @@
 @SuppressWarnings("serial")
 public class PlayerFragment extends Fragment implements IPlayerListener, Serializable, AudioManager.OnAudioFocusChangeListener, PlayerController.ShareVideoListener {
 
+    private enum VIDEO_NOT_PLAY_TYPE { isClear, isVideoMessageDisplayed, isVideoOnlyOnWeb, isNetworkMessageDisplayed, isShownWifiSettingsMessage }
```

Comment:
I think the standard java style for enums is that the name of the enum is CamelCase and the items are ALL_CAPS. So this would be:
```
enum VideoNotPlayingMessage {
IS_CLEAR, IS_VIDEO_MESSAGE_DISPLAYED, ...
```